src/generateFixedSizeNumber.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+function generateFixedSizeNumber(length) {
+  let digits = '0123456789';
+  let number = '';
+
+  for (let i = 0; i < length; i++) {
+    const randomIndex = Math.floor(Math.random() * digits.length);
+
+    number += digits[randomIndex];
+    digits = digits.slice(0, randomIndex) + digits.slice(randomIndex + 1);
+  }
```

Comment:
Would be better if you will generate your number like this `const randomDigit = Math.floor(Math.random() * 10);
`
In this case you don't need additional variable and operations with `slice`, but your solution is interesting 👍